.travis.yml

```diff
@@ -19,13 +19,15 @@ branches:
 
 before_install:
   - sudo apt-get -qq update
-  - sudo apt-get install clang-format-3.9
 
 install:
   - pip install -r requirements.txt
   - wget https://github.com/google/protobuf/releases/download/v3.4.0/protoc-3.4.0-linux-x86_64.zip
   - unzip protoc-3.4.0-linux-x86_64.zip
   - export PATH="${TRAVIS_BUILD_DIR}/bin:$PATH"
+  - wget http://releases.llvm.org/5.0.1/clang+llvm-5.0.1-x86_64-linux-gnu-ubuntu-14.04.tar.xz
+  - tar -xJf clang+llvm-5.0.1-x86_64-linux-gnu-ubuntu-14.04.tar.xz
+  - export PATH="${TRAVIS_BUILD_DIR}/clang+llvm-5.0.1-x86_64-linux-gnu-ubuntu-14.04/bin:$PATH"
```

Comment:
We should be able to cache this as well at some point, in case LLVM craps out again :)